energy-integration/farm-boosted-yields/src/lib.rs

```diff
@@ -99,13 +154,24 @@ pub trait FarmBoostedYieldsModule:
 
 pub struct FarmBoostedYieldsWrapper<T: FarmBoostedYieldsModule> {
     pub current_farm_token_nonce: Nonce,
+    pub user_farm_amount: BigUint<<T as ContractBase>::Api>,
+    pub total_farm_supply: BigUint<<T as ContractBase>::Api>,
+    pub total_rewards_per_block: BigUint<<T as ContractBase>::Api>,
     pub phantom: PhantomData<T>,
```

Comment:
I think you don't need the phantom data anymore, as you're using `T`, although I might be wrong.

Reply:
Indeed, it doesn't. 
Deleted.